forc/src/cli/commands/plugins.rs

```diff
@@ -11,33 +11,68 @@ pub struct Command {
     /// Prints the absolute path to each discovered plugin.
     #[clap(long = "paths", short = 'p')]
     print_full_path: bool,
+    /// Prints the long description associated with each listed plugin
+    #[clap(long = "describe", short = 'd')]
+    describe: bool,
 }
 
 pub(crate) fn exec(command: PluginsCommand) -> Result<()> {
-    let PluginsCommand { print_full_path } = command;
+    let PluginsCommand {
+        print_full_path,
+        describe,
+    } = command;
 
+    println!("Installed Plugins:");
     for path in crate::cli::plugin::find_all() {
-        print_plugin(path, print_full_path);
+        print_plugin(path, print_full_path, describe);
     }
     Ok(())
 }
 
+fn parse_description_for_plugin(plugin: &str) -> String {
+    use std::process::Command;
+    let proc = Command::new(plugin)
+        .arg("-h")
+        .output()
+        .expect("Could not get plugin description.");
+
+    let stdout = String::from_utf8_lossy(&proc.stdout);
+    stdout
+        .split('\n')
+        .nth(1)
+        .unwrap_or("No description found for this plugin.")
+        .to_owned()
+}
+
+fn format_print_description(path: PathBuf, print_full_path: bool, describe: bool) -> String {
```

Comment:
Not sure which functions need docstrings vs which don't. Didn't add one to these since they're private - but happy to go back and add a docstring

Reply:
Yeah for non public functions then it's at the discretion of the dev if they do this. I'd say in general its good practice to document each function so it's easier for other developers to understand when they come across it.